src/components/capture/Create/index.tsx

```diff
@@ -148,64 +150,70 @@ function CaptureCreate() {
-                    <EntityCreate
-                        title="browserTitle.captureCreate"
-                        entityType={entityType}
-                        draftSpecMetadata={draftSpecsMetadata}
-                        resetState={resetState}
-                        errorSummary={
-                            <ValidationErrorSummary
-                                errorsExist={detailsFormErrorsExist}
-                            />
-                        }
-                        toolbar={
-                            <EntityToolbar
-                                GenerateButton={
-                                    <CaptureGenerateButton
-                                        entityType={entityType}
-                                        disabled={!hasConnectors}
-                                        callFailed={helpers.callFailed}
-                                        postGenerateMutate={mutateDraftSpecs}
-                                        createWorkflowMetadata={{
-                                            initiateDiscovery,
-                                            setInitiateDiscovery,
-                                        }}
-                                    />
-                                }
-                                TestButton={
-                                    <EntityTestButton
-                                        closeLogs={handlers.closeLogs}
-                                        callFailed={helpers.callFailed}
-                                        disabled={!hasConnectors}
-                                        logEvent={CustomEvents.CAPTURE_TEST}
-                                    />
-                                }
-                                SaveButton={
-                                    <EntitySaveButton
-                                        closeLogs={handlers.closeLogs}
-                                        callFailed={helpers.callFailed}
-                                        disabled={!draftId}
-                                        materialize={
-                                            handlers.materializeCollections
-                                        }
-                                        logEvent={CustomEvents.CAPTURE_CREATE}
-                                    />
-                                }
-                            />
-                        }
-                        RediscoverButton={
-                            <RediscoverButton
-                                entityType={entityType}
-                                disabled={!hasConnectors}
-                                callFailed={helpers.callFailed}
-                                postGenerateMutate={mutateDraftSpecs}
-                            />
-                        }
-                    />
-                </ResourceConfigHydrator>
-            </EndpointConfigHydrator>
+            <DetailsFormHydrator>
```

Comment:
Addition of a details form store hydrator. No additional changes were made to the remainder of this file.